purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -48,20 +49,19 @@ import java.util.concurrent.TimeUnit
  * guide to setup your RevenueCat account.
  * @warning Only one instance of Purchases should be instantiated at a time!
  */
+@Suppress("LongParameterList")
```

Comment:
[NOT FOR THIS PR] strong sign that purchases does too much. When we get the chance, we should extract some of its responsibilities.

Reply:
Totally agree, I added it because everytime we touch this constructor detekt complains, and it's something we are already aware of.